plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1119,7 +1119,7 @@ elementgui.dimension.event_player_enters=When player enters dimension
 elementgui.dimension.event_player_leaves=When player leaves dimension
 elementgui.dimension.event_can_make_portal=Can make portal
 elementgui.dimension.event_can_travel_through_portal=Can travel through portal
-elementgui.dimension.world_gen_type=      World gen type:
+elementgui.dimension.world_gen_type=World gen type:

```

Comment:
Spaces here are for formatting reasons, also for all changes like this below. Please remove all these changes as they make parts of UI strange, also they are not part of this PR.

Reply:
I don't know why they have been removed because I never changed them. :/

Reply:
Well, they changed, so please revert the changes. Maybe use texts from master for origin and add needed lines there.